config/initializers/secure_headers.rb

```diff
@@ -44,8 +44,13 @@
   config.cookies = {
     secure: true, # mark all cookies as "Secure"
     httponly: true, # mark all cookies as "HttpOnly"
+    # We need to set the SameSite setting to "Lax", not "Strict" due to a bug
+    # in Chrome that resets the session in the new browser tab that opens when
+    # the email confirmation link is clicked. Resetting the session means losing
+    # all the SP info we stored there, meaning during account creation, a user
```

Comment:
what about passing the `request_id`in the URL params and deriving the SP info from that? We are already doing that in some places, figure it's possible to use the same technique to preserve SP info outside of session?